aips/staking-contract-to-delegation-pool.md

```diff
@@ -0,0 +1,86 @@
+---
+aip:
+title: Direct conversion from staking contract to delegation pool
+author: alexfilip2/alexandru@bwarelabs.com
+discussions-to (*optional):
+Status: Draft
+last-call-end-date (*optional):
+type: Standard (Framework)
+created: 02/04/2024
+updated (*optional):
+requires (*optional):
+---
+
+# AIP-X - Direct conversion from staking contract to delegation pool
+
+## Summary
+
+The legacy staking contract cannot be directly converted to a delegation pool. The staker should unlock their entire stake from the staking contract and then delegate it to a newly created delegation pool.
```

Comment:
Could you explain why we want to convert in the first place?

Reply:
the main reason is listed in 'Motivation' section